src/views/layouts-default/TheAccessWalletLayout.vue

```diff
@@ -126,30 +162,71 @@ export default {
             this.$router.push({ name: 'MobileApps' });
           }
         },
+        /* Software */
         {
           color: 'outline',
           title: 'Software',
           subtitle: 'Keystore files, Mnemonic phrase, Private key',
-          note: 'NOT RECOMMANDED',
+          note: 'NOT RECOMMENDED',
           rightIcon: '',
           titleIcon: 'mdi-alert',
           titleIconClass: 'warning--text text--darken-1',
           fn: () => {
-            this.open('showSoftware');
+            this.open(VALID_OVERLAYS[2]);
```

Comment:
same here, lets switch it to an Object

Reply:
fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2846/commits/2f678616ff35a0e9dccfbdce859132cf5e069d58